log/src/test/java/kcl/teamIndexZero/traffic/log/Log_ConfigTest.java

```diff
@@ -1,26 +1,53 @@
 package kcl.teamIndexZero.traffic.log;
 
 import junit.framework.TestCase;
+import kcl.teamIndexZero.traffic.log.outputs.Output_TERM;
+import kcl.teamIndexZero.traffic.log.outputs.Output_TXT;
+import org.junit.After;
+import org.junit.Before;
 import org.junit.Test;
 
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
 /**
- * Created by Alwlyan on 28/01/2016.
+ * Created by Es on 28/01/2016.
  */
 public class Log_ConfigTest extends TestCase {
+    private Log_Config config = null;
 
-    @Test
-    public void testGetFileName() throws Exception {
-        Log_Config config = new Log_Config();
-        System.out.println(config.getFileName());
+    @Before
+    public void setUp() throws Exception {
+        config = new Log_Config();
+    }
+
+    @After
+    public void tearDown() throws Exception {
+        config = null;
     }
 
     @Test
     public void testGetGlobalLogLevel() throws Exception {
+        assertEquals(config.getGlobalLogLevel(), Log_Levels.WARNING); //Default value
 
+        Log_Config mocked_config = mock(Log_Config.class);
+        when(mocked_config.getGlobalLogLevel()).thenReturn(Log_Levels.ERROR);
+        assertEquals(mocked_config.getGlobalLogLevel(), Log_Levels.ERROR);
```

Comment:
Looks like here you are exactly testing functionality of Mockito :8ball: 

The first line though belongs here totally.